AppleShopDZ/AppleShopDZ/PDFViewController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  PDFViewController.swift
+//  AppleShopDZ
+//
+//  Created by angelina on 12.10.2022.
+//
+
+import UIKit
+import WebKit
+
+/// Контроллер вызова ПДФ файла
+class PDFViewController: UIViewController, WKUIDelegate, WKNavigationDelegate {
```

Comment:
```suggestion
final class PDFViewController: UIViewController,  {
```
WKUIDelegate, WKNavigationDelegate  в extension